### PR TITLE
PLAT-22542: Change the objectCreated method to be more modular

### PR DESCRIPTION
**Pull request by shamamayair**

### CODE REVIEW

File: alpha/apps/kaltura/lib/kObjectCreatedHandler.php [hunk 1]
```diff
@@ -28,46 +28,61 @@ public function shouldConsumeCreatedEvent(BaseObject $object)
...
-		{
-			KalturaLog::info("Live entry with id [{$object->getRootEntryId()}] not found, categories will not be copied");
-			return true;
-		}
-		
-		/* @var $liveEntry LiveEntry */
-		$recordingOptions = $liveEntry->getRecordingOptions();
-		if(!$recordingOptions)
-			return true;
+        if ($object instanceof Entry)
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/kObjectCreatedHandler.php:36]: Entry is actually entry (that actual class name)

File: alpha/apps/kaltura/lib/kObjectCreatedHandler.php [hunk 2]
```diff
@@ -28,46 +28,61 @@ public function shouldConsumeCreatedEvent(BaseObject $object)
...
 	{
-		if($object instanceof KuserKgroup)
-		{
-			$kgroup = kuserPeer::retrieveByPK($object->getKgroupId());
-			$numberOfUsersPerGroup = $kgroup->getMembersCount();
-			$kgroup->setMembersCount($numberOfUsersPerGroup+1);
-			$kgroup->save();
-			return true;
-		}
+        if ($object instanceof KuserKgroup)
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/kObjectCreatedHandler.php:31]: Indentation - keep using tabs as the existing code (here space are used)

File: alpha/apps/kaltura/lib/kObjectCreatedHandler.php [hunk 3]
```diff
@@ -28,46 +28,61 @@ public function shouldConsumeCreatedEvent(BaseObject $object)
...
+        {
+            $this->handleKuserKgroupCreated($object);
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/kObjectCreatedHandler.php:33]: to keep full backward compatibility, should return when handling specific object type

File: alpha/apps/kaltura/lib/kObjectCreatedHandler.php [hunk 4]
```diff
@@ -28,46 +28,61 @@ public function shouldConsumeCreatedEvent(BaseObject $object)
...
+    protected function handleKuserKgroupCreated (KuserKgroup $object)
+    {
+        $kgroup = kuserPeer::retrieveByPK($object->getKgroupId());
+        $numberOfUsersPerGroup = $kgroup->getMembersCount();
+        $kgroup->setMembersCount($numberOfUsersPerGroup+1);
+        $kgroup->save();
+        return true;
+    }
+
+    protected function handleEntryCreated (Entry $object)
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/kObjectCreatedHandler.php:53]: object type is entry

**Review by david-winder-kaltura — APPROVED** [review]

**Pull request merged**